kagglesdk/kernels/services/kernels_api_service.py

```diff
@@ -77,3 +78,17 @@ def get_kernel_session_status(self, request: ApiGetKernelSessionStatusRequest =
       request = ApiGetKernelSessionStatusRequest()
 
     return self._client.call("kernels.KernelsApiService", "ApiGetKernelSessionStatus", request, ApiGetKernelSessionStatusResponse)
+
+  def download_kernel_output(self, request: ApiDownloadKernelOutputRequest = None) -> HttpRedirect:
```

Comment:
similar comment as above; if we don't need this in Kaggle API, maybe it would be nice to exclude it.

Reply:
This is required by `kaggle kernels output`.